Dockerfile.amd64

```diff
@@ -1,11 +1,17 @@
 FROM golang:1.13.8 as builder
 
-COPY . /go/src/github.com/oecp/rama
-
 ENV ARCH=amd64
 
 WORKDIR /go/src/github.com/oecp/rama
 
+COPY go.mod ./go.mod
+COPY go.sum ./go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
```

Comment:
what brings a re-download problem here? I even don't think there it is.

Reply:
It happens when you are developing. At that time, re-download go module is  time-consuming. And this little trick can help speed the build of image.